src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
It's not about what we support (which changes over time), it's about where the problematic behavior exists (which doesn't, except regarding not-yet-released versions).